db/seeds/post.rb

```diff
@@ -0,0 +1,622 @@
+    authors_locked: true
+  },
+  {
+    board_id: 1,
+    user_id: 2,
+    subject: "for my birthday i become a real person",
+    content: "The eighteenth birthday is the best one.  Bella is no longer A Minor.  She no longer needs Charlie's or Renée's permission to do random things that real people are allowed to do.  She still can't drink, but she doesn't feel like throwing her cognitive edge out a window anyway.<br><br>Also, she is going to get a cake.",
+    character_id: 31,
+    icon_id: 42,
+    created_at: "2012-10-12 16:02:00",
+    updated_at: "2012-10-12 21:18:00",
+    status: 1,
+    section_id: 3,
+    section_order: 5,
+    last_user_id: 2,
+    edited_at: "2012-10-12 16:02:00",
+    tagged_at: "2012-10-12 21:18:00",
+    authors_locked: true
+  },
+  {
+    board_id: 1,
+    user_id: 2,
+    subject: "style substance subterfuge",
+    content: "Miles-toting is a reasonably fun part of Elli's job.  She is privy to such secrets as \"where he is being toted from\" and \"who he will be when picked up to be toted\",
+    so she's the one who fetches Miles Vorkosigan, watches over his transformation into Miles Naismith, and accompanies the latter to his destination, pretending this time to be business partners (she Sales And Marketing, he eccentrically Development, product Top Secret).<br><br>Currently they're sliding back into their ranks and uniforms - she's got her Dendarii pants on already and lovely scarlet accessories to turn them into shiny silver fashion statements.  But before they fully reintegrate with their outfit and she dons the rest of <em>her</em> outfit there is a hospital visit to make.<br><br>The survivors of the incident, such as they are, aren't in great shape.  Miles knows how the dead-and-irrecoverable wanted to be disposed of; he knows his obligations to the crippled, likewise, and he discharges them with all the gravity and generosity that could be wished.  Quinn wants his job, so she watches him do it, even this least lovely part.  Reminds her a little of the news that her face was getting artistically replaced.  But that doesn't mean she doesn't wonder why he has to get that up close and personal with Aziz, who after all probably can't understand a word he says.<br><br>Miles demurs with something about a fascinated fear of the loss of mind, and Elli lets it go.<br><br>They get good news about Marilac, about the value of their work there.<br><br>They split up so that he can shower, and she can get into her greys, and then there's going to be the usual meeting.",
+    created_at: "2015-07-22 17:44:00",
+    updated_at: "2019-06-22 18:40:31",
+    status: 2,
+    section_order: 0,
+    last_user_id: 3,
+    edited_at: "2019-06-22 18:40:31",
+    tagged_at: "2015-07-23 00:59:00",
+    last_reply_id: 164,
+    authors_locked: true
+  },
+])
+
+puts "Assigning users to threads..."
+
+ActiveRecord::Base.connection.execute("TRUNCATE TABLE post_authors RESTART IDENTITY")
+PostAuthor.create!([
```

Comment:
Does creating replies after these exist muck with the timestamps etc at all?

Reply:
Not that I recall noticing when I repeatedly dumped them, which is weird, but it might be going with the replies created_at to check if they need updated